cms/wizards/wizard_base.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+
+from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import ImproperlyConfigured
+from django.forms.models import ModelForm
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import (
+    override as force_language,
+    force_text,
+    ugettext as _
+)
+
+
+class WizardBase(object):
+    template_name = None
+
+    def __init__(self, title, weight, form, model=None, template_name=None,
```

Comment:
Because it isn't a position. Its a sorting weight. I considered `sort_metric`, but didn't feel like it was worth changing from the source code I started with.